### DISCUSSION TITLE
Where can we find non-VSCode CSS themes for Theia?

### DISCUSSION THREAD

**Discussion by trusktr** [eclipse-theia/theia#12273] [Q&A]
VS Code themes are extremely limited and do not offer very great styling abilties.

Atom (now [Pulsar Editor](https://pulsar-edit.dev/)) provides much more interesting themes because themes in Atom (now Pulsar) can provide any CSS overrides they wish, truly customizing the look and feel of the editor.

Where can we explore non-VSCode CSS themes for Theia to really make style take off compared to highly-limited VS Code themes?

**Comment by vince-fugnitto** [comment]
@trusktr thank you for the discussion, the framework currently only support vscode themes (through extensions) in order to actually theme the application (which is based on what colors the application displays for components and UI elements).

I'm not aware of what 'pulsar editor' permits as theming but if you want additional flexibility you can always include your own styling to your application through CSS but this is quite different than allowing your end-users the ability to provide themes that drastically change the look-and-feel of the app.
